webview/src/shared/style.scss

```diff
@@ -5,14 +5,14 @@
 
 html,
 body,
-.react-root {
+#root {
```

Comment:
Changing to an ID selector so we don't have to set the class name on the element